test/ranges.jl

```diff
@@ -838,3 +838,10 @@ let r = linspace(-big(1.0),big(1.0),4)
     @test isa(@inferred(r[2]), BigFloat)
     @test r[2] ≈ big(-1.0)/3
 end
+
+let r = linspace(1.0, 3+im, 4)
```

Comment:
Fortunately, there's always `git blame`:
```julia
$ git blame --annoyed -L 842 ranges.jl
32e875ef (Tim Holy 842) let r = linspace(1.0, 3+im, 4)     # Sheesh, you'd think...
32e875ef (Tim Holy 843)     @test r[1] === 1.0+0.0im       # ...he'd have learned by now.
32e875ef (Tim Holy 844)     @test r[2] ≈ (5/3)+(1/3)im     # Do you realize how many...
32e875ef (Tim Holy 845)     @test r[3] ≈ (7/3)+(2/3)im     # ...hours this bug cost me?
32e875ef (Tim Holy 846)     @test r[4] === 3.0+im          # I'm sending Tim Holy the bill!
32e875ef (Tim Holy 847) end                                # Loser.
tim@diva:~/src/julia-0.6/test$
```
The April Fool's edition has additional fun options, like `--passive-aggressive`, `--your-boss-is-on-the-phone`, and `--rap-battle`.